Initium-Odp/src/com/universeprojects/miniup/server/commands/framework/Command.java

```diff
@@ -75,6 +77,29 @@ protected Long tryParseId(Map<String,String> params, String fieldName)
 			throw new RuntimeException(this.getClass().getSimpleName()+" invalid call format, '"+fieldName+"' is not a valid id.");
 		}
 	}
+	
+	/** 
+	 * Tries to fetch fieldName from param list and throws if it cannot.
+	 * 
+	 * @param params
+	 * @param fieldName
+	 * @param delimitingCharacter
+	 * @return List of longs, parsed from param string
+	 */
+	protected List<Long> tryParseStringToList(Map<String,String> params, String fieldName, String delimitingCharacter)
```

Comment:
It's a bit odd to do this only for a list of Long's without indicating it is for longs in the method name. Overloads will be needed for other types and string is the most obvious candidate for the non specific method name like this. Maybe just add the word Long before List?